app/models/manageiq/providers/kubernetes/inventory/parser/container_manager.rb

```diff
@@ -853,10 +853,10 @@ def parse_container_state(state_hash)
     return {} if state_hash.to_h.empty?
     res = {}
     # state_hash key is the state and value are attributes e.g 'running': {...}
-    (state, state_info), = state_hash.to_h.to_a
+    (state, state_info), = state_hash.to_h.deep_symbolize_keys.to_a
     res[:state] = state
     %w(reason started_at finished_at exit_code signal message).each do |attr|
```

Comment:
Maybe not for this PR, but we can pre-symbolize everything and not need to the .to_sym calls - Something like:

```suggestion
    %i(reason reason started_at startedAt finished_at finishedAt exit_code exitCode signal signal message message).each_slice(2) do |attr, state_info_attr|
```

Reply:
yea, instead of each_slice, just using a hash is easier for me to read. But both work

Reply:
Done